packages/forms/stories/json/typeahead.json

```diff
@@ -0,0 +1,39 @@
+{
+  "jsonSchema": {
+    "title": "A simple typeahead",
+    "description": "A simple typeahead widget example.",
+    "type": "object",
+    "properties": {
+      "simpleTypeahead": {
+        "type": "string",
+        "title": "Typeahead",
+        "inputProps": {
+          "value": "a"
+        },
+        "items": [
```

Comment:
This is not jsonschema complient, 
Take time to read this [resource](https://spacetelescope.github.io/understanding-json-schema/)